src/scripts/self-hosted-runners/setup-self-hosted-runner-ubuntu.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Check if script is run as root
+if [ "$EUID" -eq 0 ]; then
+    echo "Please run this script as a non-root user."
+    exit 1
+fi
```

Comment:
no need for that as we added it in the make file

Reply:
removed